src/Form/FormAnalyzer.js

```diff
@@ -146,7 +147,7 @@ class FormAnalyzer {
             // These are explicit signals by the web author, so we weigh them heavily
             this.updateSignal({
                 string: el.getAttribute('autocomplete') || '',
-                strength: 20,
+                strength: 10,
```

Comment:
We keep finding forms with the wrong damn attribute for `new-password` vs `current-password`. Baffling. Decreasing the signal strength.